.github/workflows/functional.yml

```diff
@@ -35,29 +35,16 @@ jobs:
             experimental: true
 
     steps:
-      - name: Checkout
-        uses: actions/checkout@v4
+      - uses: actions/checkout@v4
 
-      - name: Install PHP
-        uses: shivammathur/setup-php@v2
+      - uses: shivammathur/setup-php@v2
         with:
-          coverage: xdebug
-          php-version: ${{ matrix.php-version }}
+          php-version: "${{ matrix.php-version }}"
+          coverage: none
```

Comment:
This causes all functional tests to be skipped! Should be `coverage: xdebug`

Reply:
Sorry missed that in my copy pasta party.. Fixed